core/src/main/scala/kafka/controller/TopicDeletionManager.scala

```diff
@@ -292,22 +291,35 @@ class TopicDeletionManager(config: KafkaConfig,
    * @param replicasForTopicsToBeDeleted
```

Comment:
nit: not from this patch, but can we document or remove this `@param`

Reply:
Ack. Removed.